pefile.py

```diff
@@ -2660,21 +2661,25 @@ def parse_symbol_table(self):
             symbol_size = Structure(self.__COFF_SYMBOL_format__).sizeof()
             self.string_table_offset = self.FILE_HEADER.PointerToSymbolTable + self.FILE_HEADER.NumberOfSymbols * symbol_size
 
-            if self.string_table_offset + 4 >= len(self.__data__):
+            self.symbols = []
+            self.strings = {}
+
+            if (self.string_table_offset + 4) >= len(self.__data__):
                 self.__warnings.append('Symbol table is corrupt')
                 return
 
             string_table_size = struct.unpack('<I', self.__data__[self.string_table_offset : self.string_table_offset + 4])[0]
```

Comment:
This is the DWORD value that can apparently be invalid in some PE files. Loaded one of the samples in IDA pro and it complained of the same value for the string value size.